unicodetools/src/main/java/org/unicode/xml/UCDPropertyDetail.java

```diff
@@ -755,61 +839,69 @@ public class UCDPropertyDetail {
                     true,
                     false,
                     false,
-                    true);
-    public static UCDPropertyDetail Expands_On_NFC_Detail =
-            new UCDPropertyDetail(
-                    UcdProperty.Expands_On_NFC,
-                    VersionInfo.getInstance(3, 2, 0),
-                    84,
                     true,
-                    false,
-                    false,
-                    true);
-    public static UCDPropertyDetail Expands_On_NFD_Detail =
+                    false);
+// Deprecated
```

Comment:
i suggest you just delete this altogether

Reply:
I screwed up here - I should have added values for maxVersion, which I have done now.